scripts/update/04-chmod.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Ensures all scrips has permissions so that they are executable
+
+echo_info "Setting permissions of scripts"
+# Set permissions of all scripts
+chmod -R 700 /etc/swizzin/scripts
+
+echo_info "Setting permissions of sources"
+# Sources are only `.` or `source`d and does not need to be executable
+chmod -R 600 /etc/swizzin/sources
```

Comment:
I think this will take a couple  miliseconds, an info is too prominent in this instance and we can slap them all under one "go"
```suggestion
echo_progress_start "Checking permissions on swizzin files"

chmod -R 700 /etc/swizzin/scripts # Set permissions of all scripts
chmod -R 600 /etc/swizzin/sources # Sources are only `.` or `source`d and does not need to be executable

echo_progress_done "Permissions changed"
```

Reply:
we might need `+rx` on directories under `sources/`, e.g. `sources/functions`.